frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/EditIconModal/_styles.scss

```diff
@@ -0,0 +1,70 @@
+.edit-icon-modal {
+  h2 {
+    font-size: $medium;
+    font-weight: $regular;
+  }
+
+  &__preview-card {
+    max-height: 203px;
+    overflow: hidden;
+
+    &__fleet,
+    &__self-service {
+      width: 500px;
+    }
+
+    &__self-service {
+      height: 350px;
+
+      img {
+        width: 160%;
+      }
+    }
+  }
+
+  &__preview-table {
+    width: 520px;
+    height: 100px;
+    margin-bottom: -32px;
```

Comment:
why the negative margin?

Reply:
I think originally it was to move up the overlay but unneeded, ty